src/views/Housing/components/RDView/components/MobileView/index.jsx

```diff
@@ -11,6 +11,7 @@ import {
   Paper,
 } from '@mui/material';
 import { fetchOnDutyData } from 'services/residentLife/RA_OnCall';
+import ScottieMascot from 'views/Housing/ScottieMascot.png';
```

Comment:
Housing was updated to ResLife so this will have to get updated accordingly once merged.